Description: Comment out backports by default in sources.list
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2019-10-16

--- cloud-init-19.2.orig/templates/sources.list.debian.tmpl
+++ cloud-init-19.2/templates/sources.list.debian.tmpl
@@ -26,5 +26,5 @@ deb-src {{mirror}} {{codename}}-updates
 ## N.B. software from this repository may not have been tested as
 ## extensively as that contained in the main release, although it includes
 ## newer versions of some applications which may provide useful features.
-deb {{mirror}} {{codename}}-backports main
-deb-src {{mirror}} {{codename}}-backports main
+#deb {{mirror}} {{codename}}-backports main
+#deb-src {{mirror}} {{codename}}-backports main
